support/get-logs.sh

```diff
@@ -10,54 +10,123 @@
 # disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 #
 
-PROGRAM_NAME=$0
-VERSION="2020.1.3"
+PROGRAM_NAME="${0}"
+VERSION="2020.1.4"
 DATE=`date +%d-%m-%y`
 TIME=`date +%H-%M-%S`
 
+####################################################################################################
+# Script helper functions
+####################################################################################################
+
 usage() {
-    printf "$PROGRAM_NAME v${VERSION}\n\n"
+    printf "${PROGRAM_NAME} v${VERSION}\n\n"
     printf "This script collects the log files from available pods and tars them up.\n"
     printf "It uses the component label names instead of the pod names as the pod names could be truncated.\n\n"
-    printf "usage: $PROGRAM_NAME [-h] [-n|-ns|-namespace=NAMESPACE] [-r|-rel|-release=RELEASE]\n\n"
+    printf "usage: ${PROGRAM_NAME} [-h] [-n|-ns|-namespace=NAMESPACE] [-r|-rel|-release=RELEASE]\n\n"
     printf "  -h                       display help\n"
     printf "  -n | -ns  | -namespace   specify the required NAMESPACE\n"
     printf "  -r | -rel | -release     specify the required RELEASE\n\n"
+    printf "  -s | -since              specify how many hours back the logs should go (default 120)n"
```

Comment:
Maybe a personal preference but I think the semantics of `since` don't fit. It's more of a `last` than a `since`.

That default is definitely 120 Hours yeah?

Reply:
I used since as that is the name of the option on kubectl get logs i.e. 
`kubectl get logs <pod> --since=10h`

set to 120h i.e. 5days as you cannot specify days (e.g. you can't do 5d) and 5days was arrived at through discussion with Emma

Reply:
OK cool. Customer logs can be retained for whatever arbitrary time they choose but 5 days is probably sensible and we can let customers know about this argument if they need something older or only need to give us updated logs from the last X hours.